ca/ca.go

```diff
@@ -23,7 +23,12 @@ import (
 )
 
 type NewOpts struct {
-	Issuer       mtc.RelativeOID
+	Issuer mtc.RelativeOID
+
+	// ServerPrefix is the URL at which the CA can be reached excluding the
+	// protocol (e.g., "https://"). For example, if the server prefix is
+	// "ca.example.com/path", then the issuance endpoint is
+	// "https://ca.example.com/path/ca/queue".
 	ServerPrefix string
```

Comment:
Reviewer note: I didn't check if adding "https://" is valid.

Reply:
Nit: "excluding the implied `https://`"

Reply:
Done.